Airports/EGMC/SMR/Regions.txt

```diff
@@ -1,13 +1,12 @@
-;Blue_Background
+; - Blue_Background
```

Comment:
Just to tidy up, could you find and replace all of the ' - ' entries with nothing?
So ; - Blue_Background becomes ;Blue_Background?

Cheers! (Sorry, OCD talking)

Reply:
All done - also added some spacing between the sections so it's a little easier to navigate. Find and replace is a beautiful thing ;)